builtin_interfaces/Time stamp

dynamixel_handler/DynamixelOptionMode status

uint16[] id_list

float64[] pwm_percent
float64[] current_ma
float64[] velocity_deg_s
float64[] position_deg
float64[] vel_trajectory_deg_s
float64[] pos_trajectory_deg
float64[] input_voltage_v
float64[] temperature_deg_c